lib/livebook/runtime/evaluator/doctests.ex

```diff
@@ -17,13 +17,19 @@ defmodule Livebook.Runtime.Evaluator.Doctests do
     case define_test_module(modules) do
       {:ok, test_module} ->
         if test_module.tests != [] do
+        # grab doctests line numbers from:
+        # get_in(test_module.tests,
+        #        Access.all(),
+        #        tags,
+        #        :doctest_line)
           tests =
             test_module.tests
             |> Enum.sort_by(& &1.tags.doctest_line)
             |> Enum.map(&run_test/1)
 
           formatted = format_results(tests)
           put_output({:text, formatted})
+          # put_output({:doctest_results, doctest_results})
```

Comment:
this would be the line which initiates the signal that should arrive at the editor to indicate doctest line numbers